### PR TITLE
Updates for Frequency release candidate

### LINKED ISSUE #13: Update to use latest frequency api-augment release
Once a release is cut for [Frequency](https://github.com/LibertyDSNP/frequency), update this repo to use the latest npm package for it.

### PR DESCRIPTION
**Pull request by shannonwells**
The purpose of this PR is to update a bunch of stuff for the Frequency release. Closes #13 

Specifically:

- [x] Update to latest `@dsnp/api-augment` package for Frequency
- [X] Update code to reflect latest changes:
  - [x] Provider must also be registered as well as having an MSA created
  - [x] Update sponsored MSA creation: change `DelegateData` and `scaleEncodeDelegateData` for `createSponsoredAccountWithDelegation`
  - [x] Create schema works again
  - [x] Fetch schemas works again
  - [x] Creating messages works again

Additionally:
- [x] Tweaked some error behavior and messaging
- [x] Updated other packages
- [x] Added .tool-versions and .node-version files for users of `asdf` and `nvm`
- [x] Switched from yarn to npm because yarn was causing problems for some
- [x] Other little cleanup bits (like fixing linting)

### CODE REVIEW

File: src/services/chain/apis/extrinsic.ts [hunk 1]
```diff
@@ -95,46 +110,65 @@ export const createAccountViaService = async (
...
+      .catch((error: any) => {
+        errorCallback(error);
+      });
+}
+
 export const fetchAllSchemas = async (): Promise<Array<SchemaDetails>> => {
   const api = requireGetProviderApi();
 
-  const schema_id = await (await api.query.schemas.schemaCount()).toNumber();
+  const schemasMaxHex = (await (await api.query.schemas.currentSchemaIdentifierMaximum())).toHex();
```
Comment (shannonwells) [inline src/services/chain/apis/extrinsic.ts]: `toNumber` was failing to build saying there is no such method. I just chose something easy

File: src/components/CreateSchema.tsx [hunk 2]
```diff
@@ -1,4 +1,4 @@
-import {registerSchema } from "../services/chain/apis/extrinsic";
+import {createSchema } from "../services/chain/apis/extrinsic";
```
Comment (wilwade) [inline src/components/CreateSchema.tsx]: Not wrong, but I'm surprised that linting/formatting didn't make sure it had a space. Might be worth checking to make sure linting is working.
```suggestion
import { createSchema } from "../services/chain/apis/extrinsic";
```

File: src/services/chain/apis/extrinsic.ts (hunk 1 above)
Comment (wilwade) [reply src/services/chain/apis/extrinsic.ts]: Doesn't it come back as a bigint?

File: README.md [hunk 3]
```diff
@@ -2,12 +2,11 @@
...
 This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
-But it uses yarn instead of npm.
 
 ## Get started
-* Make sure [yarn is installed](https://classic.yarnpkg.com/en/docs/getting-started)
+* Make sure [npm is installed](https://docs.npmjs.com/cli/v9/configuring-npm/install)
 * Clone this repo
-* `yarn install`
+* `npm install`
 * Follow instructions for [building and starting Frequency](https://github.com/LibertyDSNP/frequency)
```
Comment (wilwade) [inline README.md]: (This could be a separate PR if you want) Could you add a note about matching versions?

I'm also tempted to say we should just suggest connecting to docker instant seal (easier version matching as well) instead of building.

File: README.md [hunk 4]
```diff
@@ -2,12 +2,11 @@
 # Example Client for Frequency
 
 This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
-But it uses yarn instead of npm.
```
Comment (wilwade) [inline README.md:5]: 🙇 I know Parity likes yarn, but I'm glad not to be always using the wrong one.

**Review by wilwade — COMMENTED** [review]
Few small notes, but looking great. Thanks for all the work on this.

File: src/components/CreateSchema.tsx (hunk 2 above)
Comment (shannonwells) [reply src/components/CreateSchema.tsx]: Fixed the linting.  It was skipping .tsx files.

File: README.md (hunk 3 above)
Comment (shannonwells) [reply README.md]: Updated the README with this info

File: src/components/CreateSchema.tsx [hunk 5]
```diff
@@ -0,0 +1,75 @@
+import { createSchema } from "../services/chain/apis/extrinsic";
```
Comment (shannonwells) [inline src/components/CreateSchema.tsx:1]: This was renamed

File: src/services/chain/apis/extrinsic.ts (hunk 1 above)
Comment (shannonwells) [reply src/services/chain/apis/extrinsic.ts]: apparently I hadn't checked that you could call "toBigInt" but it's fixed now.

**Review by enddynayn — APPROVED** [review]
💯 nice improvements!

**Review by saraswatpuneet — APPROVED** [review]
